core/upgrade/conditions.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Elementor\Core\Upgrade;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Conditions {
+
+	public static function _v_3_15_9_container_updates__should_run() {
+		return ( 'yes' === get_option( 'elementor_container_gap_updated' ) ) ? false : true;
```

Comment:
`elementor_container_gap_updated` flag is added when the reversal process took place. And this flag is deleted again in 3.16